tools/bricks/feature/__brick__/lib/features/{{name.snakeCase()}}/service/{{name.snakeCase()}}_service.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter_template/features/{{name.snakeCase()}}/domain/repository/i_{{name.snakeCase()}}_repository.dart';
+import 'package:flutter_template/features/{{name.snakeCase()}}/service/i_{{name.snakeCase()}}_service.dart';
+
+/// Implementation of [I{{name.pascalCase()}}Repository]
+final class {{name.pascalCase()}}Service implements I{{name.pascalCase()}}Service {
+
+  final I{{name.pascalCase()}}Repository _repository;
+
+  /// Create an instance of {{name.pascalCase()}}Service
```

Comment:
It might be better to use a macro that will copy the documentation from the class. I think this should be done on all constructors in the template.